src/bootstrap/dist.rs

```diff
@@ -1282,6 +1323,7 @@ impl Step for Extended {
         let rustc_installer = builder.ensure(Rustc { compiler: builder.compiler(stage, target) });
         let cargo_installer = builder.ensure(Cargo { compiler, target });
         let rustfmt_installer = builder.ensure(Rustfmt { compiler, target });
+        let rust_demangler_installer = builder.ensure(Rustfmt { compiler, target });
```

Comment:
I think this is a mistake. Shouldn't `RustDemangler` be put here?
```suggestion
        let rust_demangler_installer = builder.ensure(RustDemangler { compiler, target });
```

Reply:
Oops. Yep... Thanks!

Reply:
fixed